Windows builds fail again

#4718 broke it

```
/home/jenkins/workspace/qTox-cmake-nightly_build_windows_x86_release/src/net/toxmedata.cpp: In member function ‘QString ToxmeData::createAddressRequest(ToxId, const QString&, const QString&, bool) const’:
/home/jenkins/workspace/qTox-cmake-nightly_build_windows_x86_release/src/net/toxmedata.cpp:123:63: error: ‘time’ was not declared in this scope
            "\",\"timestamp\":" + QString().setNum(time(nullptr)) +
                                                               ^
/home/jenkins/workspace/qTox-cmake-nightly_build_windows_x86_release/src/net/toxmedata.cpp: In member function ‘QString ToxmeData::deleteAddressRequest(const ToxPk&)’:
/home/jenkins/workspace/qTox-cmake-nightly_build_windows_x86_release/src/net/toxmedata.cpp:161:63: error: ‘time’ was not declared in this scope
            "\",\"timestamp\":" + QString().setNum(time(nullptr)) +
                                                               ^
CMakeFiles/qtox_static.dir/build.make:1267: recipe for target 'CMakeFiles/qtox_static.dir/src/net/toxmedata.cpp.obj' failed
```

[Full build log](https://build.tox.chat/job/qTox-cmake-nightly_build_windows_x86_release/18/console) (it will disappear after few more commits to master).

Sounds like

```C++
#include <ctime>
```

is missing. There might be more things missing too, hard to say since it could be just the first error the compiler stops on.

--- a/src/net/toxmedata.cpp
+++ b/src/net/toxmedata.cpp
@@ -22,6 +22,8 @@
 
 #include <QString>
 
+#include <ctime>
+
 QByteArray ToxmeData::parsePublicKey(const QString& text) const
 {
     static const QByteArray pattern{"key\":\""};

